shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -175,6 +192,31 @@ - (BOOL)clipboardHasStrings;
 
 @end
 
+#pragma mark - FlutterViewWrapper implementation.
+
+@implementation FlutterViewWrapper {
+  __weak FlutterView* _flutterView;
+}
+
+- (instancetype)initWithFlutterView:(FlutterView*)view {
+  self = [super initWithFrame:view.frame];
+  if (self) {
+    _flutterView = view;
+  }
+  return self;
+}
+
+- (void)setFrameSize:(NSSize)newSize {
```

Comment:
> Do you mean this method? https://developer.apple.com/documentation/appkit/nsview/1483281-autoresizingmask?language=objc

Correct

> I did this manually because the FlutterView has custom override of setFrameSize. I want to make sure it gets called

If FlutterView has code that needs to run on resize and it's not called in some cases where resizing is being done in standard ways, that would be a bug in FlutterView. A view subclass can't dictate which methods of its public API will be used for resizing.

What is the specific case where that method isn't called when it should be?

Reply:
you are right, I have made the change to use the autosize, and it seems to call the method correctly. let me resolve this comment